dist/pythonlibs/testrunner/utils.py

```diff
@@ -20,8 +20,24 @@ def test_utils_interactive_sync(child, retries, delay):
         if ret == 0:
             break
     else:
-        # Last call to make it fail her,
+        # Last call to make it fail here,
         child.expect_exact('READY', timeout=0)
 
     child.sendline('s')
     child.expect_exact('START')
+
+
+def test_utils_interactive_sync_shell(child, retries, delay):
+    """Synchronisation for 'test_utils_interactive_sync' function
+    when shell module is used.
+
+    Interacts through input to wait for node being ready.
+    """
+    for _ in range(0, retries):
```

Comment:
```suggestion
    for _ in range(retries):
```

Reply:
(exists in `test_utils_interactive_sync()` as well and should be piggybacked)